.github/workflows/check.yml

```diff
@@ -34,6 +34,15 @@ jobs:
           rustup update stable
           rustup target add wasm32-unknown-unknown --toolchain nightly
 
+      # Check formatting of project according to .rustfmt.toml rules
+      - name: Lint Code
+        run: |
+          rustfmt \
+          node/src/lib.rs \
+          pallets/nft/src/lib.rs \
+          pallets/kitties/src/lib.rs \
+          runtime/src/lib.rs --check
```

Comment:
`rustfmt` can't take a glob like other tools like `prettier`, you have to give one or more `.rs` files at the root, and it will traverse through each and find associated files. We have to give it more than one because it is not smart enough to traverse projects that do things like compile different crates/modules separately down to wasm (e.g. runtime and pallets).

Reply:
Why don't you use `cargo fmt -- --check` instead?

Reply:
Agreed, `cargo fmt -- --check` looks into all.

Reply:
Oh, that's a lot nicer